WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -482,6 +485,7 @@ protected void onCreate(Bundle savedInstanceState) {
         ((WordPress) getApplication()).component().inject(this);
         mDispatcher.register(this);
         mViewModel = new ViewModelProvider(this, mViewModelFactory).get(StorePostViewModel.class);
+        mStorageUtilsViewModel = new ViewModelProvider(this, mViewModelFactory).get(StorageUtilsViewModel.class);
```

Comment:
This is probably a minor optimization (not needed at all), but wondering if `new ViewModelProvider(this, mViewModelFactory)` could be called once and assigned to a local variable we could reuse? Feel free to overlook this comment if you don't think it's any better 👍

Reply:
For the record, I searched elsewhere out of curiosity so sharing this here https://www.bettercodebytes.com/the-cost-of-object-creation-in-java-including-garbage-collection/ (tl;dr we can probably just leave it as is 👍 😅 )

Reply:
>tl;dr we can probably just leave it as is 👍 😅

Agree, and thanks for sharing that interesting post 🙇